.github/workflows/commit.yaml

```diff
@@ -0,0 +1,34 @@
+name: Commit Workflow
+on: [push]
+
+jobs:
+  build:
+    name: Test & Build
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: docker://weaveworks/eksctl-build:19ff80e010d2fdb7be3c44fc6491e13395a682ad
```

Comment:
Nit: do you know if there is a way to factorise `docker://weaveworks/eksctl-build:19ff80e010d2fdb7be3c44fc6491e13395a682ad`?

Reply:
Not sure. For now `make -f Makefile.docker update-build-image-manifest` handles updating this.

Reply:
Some of the fields in github configs accept shell syntax (like `foo-$(echo 123)`), but that didn't work here.